cellmlmanip/units.py

```diff
@@ -79,6 +79,76 @@
 }
 
 
+class UnitError(Exception):
+    """ base class for unit errors
+    """
+    pass
+
+
+class InvalidUnitsError(UnitError):
```

Comment:
Better description? "Invalid" in some sense that's not already implied by "Error"